xtp/src/pyxtp/CMakeLists.txt

```diff
@@ -28,26 +18,24 @@ set_target_properties(xtp_binds PROPERTIES XTP_PYPATH "${XTP_PYPATH}")
 target_link_libraries(xtp_binds PUBLIC VOTCA::votca_xtp)
 
 if(NOT CMAKE_INSTALL_PYTHON_LIBDIR)
-  set(CMAKE_INSTALL_PYTHON_LIBDIR "${CMAKE_INSTALL_LIBDIR}/python${Python_VERSION_MAJOR}.${Python_VERSION_MINOR}/site-packages")
+  set(CMAKE_INSTALL_PYTHON_LIBDIR ${Python_SITELIB})
 endif()
-install(TARGETS xtp_binds LIBRARY DESTINATION ${CMAKE_INSTALL_PYTHON_LIBDIR}/pyxtp)
-add_library(VOTCA::xtp_binds ALIAS xtp_binds)
 
-file(GLOB_RECURSE PYXTP_FILES pyxtp/*.py)
-message(STATUS "PYTHON INSTALL: ${CMAKE_INSTALL_PYTHON_LIBDIR}")
-install(FILES ${PYXTP_FILES} DESTINATION ${CMAKE_INSTALL_PYTHON_LIBDIR}/pyxtp)
+install(TARGETS xtp_binds LIBRARY DESTINATION ${CMAKE_INSTALL_PYTHON_LIBDIR}/pyxtp_binds)
 
-# copy py files to build dir, so CMAKE_CURRENT_BINARY_DIR is a complete py module with script and xtp_binds
-foreach(_PYFILE ${PYXTP_FILES})
-  get_filename_component(_FILE "${_PYFILE}" NAME)
-  add_custom_command(OUTPUT ${CMAKE_CURRENT_BINARY_DIR}/${_FILE}
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${_PYFILE} ${CMAKE_CURRENT_BINARY_DIR}/${_FILE}
-    DEPENDS ${_DEP}
+install (CODE "
+    message(STATUS \"Installing Python package with pip from ${CMAKE_CURRENT_SOURCE_DIR}...\")
+    message(STATUS \"${Python_EXECUTABLE} -m pip install ${CMAKE_CURRENT_SOURCE_DIR}\")
+    execute_process(
+    COMMAND ${Python_EXECUTABLE} -m pip install ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
The CI fails here as pip isn't honoring `CMAKE_INSTALL_PREFIX` nor `DESTDIR`, however, I am not sure how to do that

Reply:
Ah I did forget to account for this indeed. Pip nowadays will error out if not called from a virtualenv. We could pass --break-system-packages and it will install to the root site lib. Python packages are managed separately anyhow and should not be placed in to `CMAKE_INSTALL_PREFIX`

Reply:
My fix actually increased headaches. pip allows installing to a specified directory, so changed to that.

Reply:
Lammps is using https://github.com/lammps/lammps/blob/develop/python/install.py